src/Api/Transaction/GetService.php

```diff
@@ -14,22 +14,40 @@ class GetService extends Transaction
 {
     protected $apiTokenRequired = true;
     protected $serviceIdRequired = true;
+    protected $version = 16;
+
+    /**
+     * @var int The ID of the payment method. Only the payment options linked to the provided payment method ID will be returned if an ID is provided. If omitted, all available payment options are returned. Use the following IDs to filter the options:.
+     */
+    private $paymentMethodId;
 
     /**
      * @var array cached result
      */
     private static $cache = array();
 
+    /**
+     * @param int $paymentMethodId
+     */
+    public function setPaymentMethodId($paymentMethodId)
```

Comment:
@kevinverschoor  Paymentmethod meegeven lijkt niet te werken. Ligt dit aan mij, of werkte dit bij jou wel?

Reply:
volgens ian klopt het dat dit niet werkt.